tonami/pitch_process.py

```diff
@@ -191,6 +190,29 @@ def extract_amplitude(track, window_len):
         # Therefore, these windows are NOT overlapping.
         lower = frame * window_len
         upper = (frame + 1) * (window_len) - 1
+        
```

Comment:
if these are tests, put tests in the test folder! if code is for visualization purposes, add a function to the data viz stuff
ifthis is a test for pitch_process, i think you can put it in the test_pitch_process.py file?